### PR TITLE
Add raw manifest in kustomize for csi driver 

### PR DESCRIPTION
**Pull request by avi-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/namespaces/admin/csi-secret-store-provider/kustomization.yaml [hunk 1]
```diff
@@ -1,4 +1,6 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-  - secrets-store-csi-driver.yaml
+  - https://raw.githubusercontent.com/Azure/secrets-store-csi-driver-provider-azure/master/deployment/provider-azure-installer.yaml
```
Comment (timja) [inline k8s/namespaces/admin/csi-secret-store-provider/kustomization.yaml:4]: don't use master, use released versions

Comment (adusumillipraveen) [inline k8s/namespaces/admin/csi-secret-store-provider/kustomization.yaml:4]: Must point to release tag than master as we don't want breaking changes applied .

**Review by adusumillipraveen — CHANGES_REQUESTED** [review]
Also, would be good to test in an env? just to be sure the patch u did doesnt have any impact.

**Review by avi-hmcts — COMMENTED** [review]
@adusumillipraveen As per our discussion made the changes to test the raw manifest for the csi driver and azure provider initially on the sandbox

**Pull request closed without merging**